simplesat/package.py

```diff
@@ -68,6 +71,10 @@ def version(self):
     def dependencies(self):
         return self._dependencies
 
+    def __repr__(self):
+        return "PackageMetadata('{0}-{1}', key={2!r})".format(
+            self._name, self._version, self._key)
```

Comment:
Aha. That makes sense.
On Jan 14, 2016 1:43 AM, "David Cournapeau" <notifications@github.com>
wrote:

> In simplesat/package.py
> <https://github.com/enthought/sat-solver/pull/114#discussion_r49695519>:
>
> > @@ -68,6 +71,10 @@ def version(self):
> >      def dependencies(self):
> >          return self._dependencies
> >
> > +    def __repr__(self):
> > +        return "PackageMetadata('{0}-{1}', key={2!r})".format(
> > +            self._name, self._version, self._key)
>
> Key in the original code had a completely different meaning (was the egg
> filename, keeping the casing information lost in the name)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/enthought/sat-solver/pull/114/files#r49695519>.
>